### DISCUSSION TITLE
pass fields to extraModalFooterActions

### DISCUSSION THREAD

**Discussion by Dontorpedo** [filamentphp/filament#11775] [Help]
### Package

Actions

### Package Version

v3

### How can we help you?

Hello, im trying to pass the form data to my Action in extraModalFooterActions, but it seems not to be possible, is there any workaround for it??

**Comment by GamendeBrian** [comment]
I have also been stuck on this problem. The $data parameter is empty...

**Reply by orebarranco** [reply]
Any solution for this?
